commands/show_cluster.go

```diff
@@ -181,35 +209,52 @@ func showClusterRunOutput(cmd *cobra.Command, cmdLineArgs []string) {
 		fmt.Println(color.WhiteString("Fetching details for cluster %s", args.clusterID))
 	}
 
-	clusterDetails, err := getClusterDetails(args.clusterID, showClusterActivityName)
-
-	var credentialDetails *models.V4GetCredentialResponse
-
-	if err == nil && clusterDetails.CredentialID != "" {
-		if args.verbose {
-			fmt.Println(color.WhiteString("Fetching details for credential %s", clusterDetails.CredentialID))
-		}
-
-		credentialDetails, err = getOrgCredentials(clusterDetails.Owner, clusterDetails.CredentialID, showClusterActivityName)
+	clusterDetailsChan := make(chan *models.V4ClusterDetailsResponse)
+	clusterDetailsErrChan := make(chan error)
+
+	go func(chan *models.V4ClusterDetailsResponse, chan error) {
```

Comment:
What is all this supposed to do? When your goal is to run different actions in parallel, please use https://godoc.org/golang.org/x/sync/errgroup.

Reply:
Looked at it, but found that `errgroup's` `Group.Wait` would only return the first error happening. That doesn't help me in this case here, as I want to treat an error happening when fetching cluster details (which is the critical request) different from errors when fetching cluster status.